storage/innobase/sync/sync0arr.cc

```diff
@@ -590,7 +590,7 @@ sync_array_cell_print(
 				"number of readers " ULINTPF
 				", waiters flag %u, "
 				"lock_word: " ULINTPFx "\n"
-				"Last time read locked in file %s line %u\n"
+				"Last time read locked in file %s line " ULINTPF "\n"
```

Comment:
Please use atomic load both for last_s_line and last_s_file_name. There should be 4 occurrences total.